File: backend/src/main/java/com/NBE3_4_2_Team4/global/security/oauth2/logoutService/impl/NaverLogoutService.java
package com.NBE3_4_2_Team4.global.security.oauth2.logoutService.impl;

import com.NBE3_4_2_Team4.domain.member.member.entity.Member;
import com.NBE3_4_2_Team4.global.security.oauth2.logoutService.OAuth2LogoutService;
import org.springframework.stereotype.Component;

@Component
public class NaverLogoutService extends OAuth2LogoutService {
    @Override
    public Member.OAuth2Provider getOAuth2Provider() {
        return Member.OAuth2Provider.NAVER;
    }
}

File: backend/src/main/java/com/NBE3_4_2_Team4/global/security/oauth2/logoutService/impl/NaverLogoutService.kt
package com.NBE3_4_2_Team4.global.security.oauth2.logoutService.impl

import com.NBE3_4_2_Team4.domain.member.member.entity.Member
import com.NBE3_4_2_Team4.global.security.oauth2.logoutService.OAuth2LogoutService
import org.springframework.stereotype.Service

@Service
class NaverLogoutService: OAuth2LogoutService() {
    override fun getOAuth2Provider(): Member.OAuth2Provider {
        return Member.OAuth2Provider.NAVER
    }
}